### PR TITLE
Specify blank index explicitly

### LINKED ISSUE #595: blank_index in Greedy Decoder not same in training and testing leads to different validation and test accuracy
While training, ```GreedyDecoder``` is [called](https://github.com/SeanNaren/deepspeech.pytorch/blob/master/deepspeech_pytorch/training.py#L129) with default values, which [gives](https://github.com/SeanNaren/deepspeech.pytorch/blob/78f7fb791f42c44c8a46f10e79adad796399892b/deepspeech_pytorch/decoder.py#L154) blank_index value of 0.

But while testing, ```GreedyDecoder``` is [called](https://github.com/SeanNaren/deepspeech.pytorch/blob/master/deepspeech_pytorch/testing.py#L21) with ```blank_index=model.labels.index('_')```.

This behavior is not an issue when ```labels.json``` contains the blank character (underscore: ```_```) at the first instance only. But if you are training on a different natural language and hence creating your own labels.json file then you need to put underscore in the beginning else you will observe different accuracy on the same validation set. As the trainer would have assumed that 0th position character in labels.json is blank_symbol whereas tester finds the correct index of blank_symbol (underscore).

I think this is a bug. If it is not, then please mention the guidelines to construct labels.json file when training on a different natural language.

### PR DESCRIPTION
**Pull request by piraka9011**
The `BeamCTCDecoder` assumes that the `blank_index` is always at position 0.
Specify it explicitly to avoid potential issues with labels.

Resolves #595

### CODE REVIEW

**Comment by SeanNaren** [comment]
Thanks man! We'll fix CI in a separate PR
